src/serializer.rs

```diff
@@ -43,3 +44,18 @@ impl<'a> Serializable for Node<'a> {
         }
     }
 }
+
+
+impl<'a> ToString for Node<'a> {
+    fn to_string(&self) -> String {
+        let mut utf_vec = Vec::new();
+        let result = match serialize(&mut utf_vec, self, Default::default()) {
+            Ok(_) => match String::from_utf8(utf_vec)  {
+                Ok(s) => s,
+                Err(_) => String::new(),
+            },
+            Err(_) => String::new(),
```

Comment:
Errors should not pass silently. It’s not OK to just return an empty string on `Err`. In this case, both `match` expressions can be replaced with `.unwrap()`: `serialize` only returns `Err` if `Writer::write` does, which it never does for `Vec<u8>`, and non-UTF-8 data would indicate a bug in html5ever. So it shouldn’t happen, but if it *does* then a panicking `.unwrap()` will be more helpful than an empty string to track down the error.